ta/arch/arm/link.mk

```diff
@@ -22,6 +22,10 @@ link-ldflags += --sort-section=alignment
 link-ldflags += -z max-page-size=4096 # OP-TEE always uses 4K alignment
 link-ldflags += --as-needed # Do not add dependency on unused shlib
 link-ldflags += $(link-ldflags$(sm))
+ifeq ($(CFG_TA_FTRACE_SUPPORT),y)
+$(file > dyn_list,{__ftrace_info;};)
```

Comment:
I'd rather have a proper target for this file. There is no reason it should be treated differently.

Reply:
ok